iu_xray_ontology.json

```diff
@@ -0,0 +1,53 @@
+
+{
+    "name": "iu_xray_ontology",
+    "additional_prefixes": [
+      "iu_xray.onto.iu_xray_ontology"
+    ],
+    "definitions": [
+      {
+        "entry_name": "iu_xray.onto.iu_xray_ontology.Findings",
+        "parent_entry": "forte.data.ontology.top.Generics",
+        "description": "A span based annotation class Findings, used to refer to findings part of the report",
+        "attributes": [
+          {
+            "name": "content",
```

Comment:
You won't need an additional content field if you use the span-based `Annotation`

Reply:
modified